### PR TITLE
GG-38808 Add BASELINE_NODE_ATTRIBUTES system view (#9204)

### PR DESCRIPTION
**Pull request by ygerzhedovich**
(cherry picked from commit 30c36e4e529e41d85e5dec8d2920a7c5cd9e2b54)

### CODE REVIEW

**Review by zstan — APPROVED** [review]
LGTM, overall looks good, plz check minors ?

File: docs/_docs/monitoring-metrics/system-views.adoc [hunk 1]
```diff
@@ -302,6 +302,20 @@ The NODE_ATTRIBUTES view contains the attributes of all nodes.
 
 |===
 
+== CONFIGURATION
```
[the remark this answers is not in the document]
Comment (ygerzhedovich) [reply docs/_docs/monitoring-metrics/system-views.adoc:305]: it was ported as is. But we have separate ticket to document all new views.

File: modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/IgniteConfigurationIterable.java [hunk 2]
```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2019 GridGain Systems, Inc. and Contributors.
```
[the remark this answers is not in the document]
Comment (ygerzhedovich) [reply modules/core/src/main/java/org/apache/ignite/internal/managers/systemview/IgniteConfigurationIterable.java:2]: Honestly I don't know have we such rule or not

File: modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java [hunk 3]
```diff
@@ -245,6 +247,12 @@ public class GridCacheProcessor extends GridProcessorAdapter {
     /** System view description for page lists. */
     public static final String CACHE_GRP_PAGE_LIST_VIEW_DESC = "Cache group page lists";
 
+    /** System view name for partition states. */
+    public static final String PART_STATES_VIEW = "partitionStates";
+
+    /** System view description for partition states. */
+    public static final String PART_STATES_VIEW_DESC = "Distribution of cache group partitions across cluster nodes";
```
[the remark this answers is not in the document]
Comment (ygerzhedovich) [reply modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java:254]: I think we should keep it as is to have less compatibility issues for Ignite users who migrate to GG

**Pull request merged**